src/model/orchestrator/providers/azure-aci/index.ts

```
@@ -0,0 +1,536 @@
+    secrets: OrchestratorSecret[],
+  ): Promise<string> {
+    OrchestratorLogger.log(`[Azure ACI] Running task for build ${buildGuid}`);
+    ResourceTracking.logAllocationSummary('azure-aci task');
+
+    const containerName = `unity-build-${buildGuid}`
+      .toLowerCase()
+      .replace(/[^a-z0-9-]/g, '-')
+      .slice(0, 63);
+
+    // Build environment variable flags
+    const allEnvVars = [
```

ESLint

[error] The variable `allEnvVars` should be named `allEnvironmentVariables`. A more descriptive name will do too.

(unicorn/prevent-abbreviations)